cluster/shard_mapper.go

```diff
@@ -154,10 +150,15 @@ func (r *RemoteMapper) Open() (err error) {
 
 	// Decode the first response to get the TagSets.
 	r.tagsets = r.bufferedResponse.TagSets()
+	r.fields = r.bufferedResponse.Fields()
```

Comment:
Fields went in after `RemoteMapper` was first coded. Obviously we forgot to add it to the `RemoteMapper` implementation.